cpu/cpu_darwin.go

```diff
@@ -88,5 +88,18 @@ func InfoWithContext(ctx context.Context) ([]InfoStat, error) {
 }
 
 func CountsWithContext(ctx context.Context, logical bool) (int, error) {
-	return runtime.NumCPU(), nil
+	var cpuArgument string
+	if logical {
+		cpuArgument = "hw.logicalcpu"
+	} else {
+		cpuArgument = "hw.physicalcpu"
+	}
+
+	count, err := unix.SysctlUint32(cpuArgument)
+
+	if err != nil {
```

Comment:
I think it would be better to put the 98, 100 lines together without a new line. That would be more go-ish.

Reply:
@mingrammer Thank you for your comment. However, I don't understand why those two lines are less go-ish. Could you let me know what real go-ish way is or send me any articles to help to understand more go-ish way?

Reply:
It's closer to convention rather than rules. Most Go projects including this project treating this kind of code as a single block by placing the error handling line directly underneath the expressions. So I said it would be more go-ish code.

Reply:
@mingrammer I've understood what you mean. Error handling and error assignment should be together to distinguish those codes from others, right? Good point 👍

Reply:
Yap, you're right!